README.adoc

```diff
@@ -304,13 +303,13 @@ The default hostname for minikube is {minikube-ip}. Otherwise it can be found us
 
 Then `curl` or visit the following URLs to access your microservices, substituting the appropriate hostname:
 
-- `{name-api}`
-- `{ping-api}/name-service`
+- `{system-api}`
+- `{inventory-api}/system-service`
 
-The first URL returns a brief greeting followed by the name of the pod that the `name` microservice
-runs in. The second URL returns `pong` if it received a good response from the `name-service`
-{kube} Service. Visiting `{ping-api}/[kube-service]` in general returns either
-a good or a bad response depending on whether `kube-service` is a valid {kube} Service that can be accessed.
+The first URL returns system properties and the name of the pod in an HTTP header called `X-Pod-Name`.
```

Comment:
- [ ] May need to tell how to show the headers, especially "X-Pod-Name", e.g. ` curl http://localhost:31000/system/properties -I`